hazelcast/src/test/java/com/hazelcast/mapreduce/impl/MapReduceUtilTest.java

```diff
@@ -0,0 +1,91 @@
+package com.hazelcast.mapreduce.impl;
+
+import com.hazelcast.mapreduce.impl.operation.NotifyRemoteExceptionOperation;
+import com.hazelcast.mapreduce.impl.task.JobProcessInformationImpl;
+import com.hazelcast.mapreduce.impl.task.JobSupervisor;
+import com.hazelcast.mapreduce.impl.task.JobTaskConfiguration;
+import com.hazelcast.mapreduce.impl.task.MemberAssigningJobProcessInformationImpl;
+import com.hazelcast.nio.Address;
+import com.hazelcast.spi.NodeEngine;
+import com.hazelcast.spi.OperationService;
+import com.hazelcast.spi.partition.IPartitionService;
+import com.hazelcast.test.HazelcastParallelClassRunner;
+import com.hazelcast.test.annotation.ParallelTest;
+import com.hazelcast.test.annotation.QuickTest;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import java.util.concurrent.TimeoutException;
+
+import static com.hazelcast.cluster.memberselector.MemberSelectors.DATA_MEMBER_SELECTOR;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.*;
+
+@RunWith(HazelcastParallelClassRunner.class)
+@Category({QuickTest.class, ParallelTest.class})
+public class MapReduceUtilTest {
```

Comment:
The whole MapReduce framework has been deprecated. The complete tests have been excluded in the `pom.xml`:
```xml
<exclude>**/mapreduce/**.java</exclude>
```
So I guess a new test here will not improve anything :)

Reply:
I see thanks, deleting added test.